osu.Game/Rulesets/UI/FrameStabilityContainer.cs

```diff
@@ -150,6 +152,18 @@ private void updateClock()
                     state = PlaybackState.NotValid;
             }
 
+            // This is a hotfix for https://github.com/ppy/osu/issues/26879 while we figure how the hell time is seeking
+            // backwards by 11,850 ms for some users during gameplay.
+            //
+            // It basically says that "while we're running in frame stable mode, and don't have a replay attached,
+            // time should never go backwards". If it does, we stop running gameplay until it returns to normal.
+            if (!hasReplayAttached && FrameStablePlayback && proposedTime > referenceClock.CurrentTime && !DebugUtils.IsNUnitRunning)
+            {
+                Logger.Log($"Denying backwards seek during gameplay (reference: {referenceClock.CurrentTime:N2} stable: {proposedTime:N2})");
```

Comment:
Is this going to be enough logging? As in, is it going to tell us any more than the 'playback discrepancy' messages we've already seen?

Reply:
It's not. I'm still working on some kind of logging which will be helpful. Feel free to wait for that before re-review.